checker/manual/constant-value-checker.tex

```diff
@@ -49,11 +49,31 @@ \subsection{Type Annotations\label{constant-value-checker-type-annotations}}
 The set is limited to 10 entries; if a variable
 could be more than 10 different values, the Constant Value
 Checker gives up and its type becomes
-\refqualclass{common/value/qual}{UnknownVal} instead.
+\refqualclass{common/value/qual}{UnknownVal} instead (except for
+\refqualclass{common/value/qual}{IntVal}
+, see below)
 
 % \refqualclass{checker/value/qual}{BottomVal}, meaning that the expression
 % is dead or always has the value \<null>.
 
+In the case of
+\refqualclass{common/value/qual}{IntVal}
+with more than 10 possible values, its type becomes
+\refqualclass{common/value/qual}{IntRange} instead of
+\refqualclass{common/value/qual}{UnknownVal},
+which is a type annotation with two integer arguments: \<from> and \<to>.
+It means that at run time, the expression only evaluates to a value that
+falls into the interval described by the two arguments. For example, an
+expression of type \<@IntRange(from = 0, to = 255)> evaluates to a value
+that falls between 0 and 255.
+
+The introduction of \<@IntRange> gives more granularity for the propagation
```

Comment:
"more granularity" isn't proper English.  If you mean it's more fine-grained, say that.  Granularity can be small or large.